sepal_ui/sepalwidgets/app.py

```diff
@@ -461,12 +482,25 @@ def add_banner(self, msg, **kwargs):
         kwargs["prominent"] = kwargs.pop("prominent", True)
         kwargs["dismissible"] = kwargs.pop("dismissible", True)
         kwargs["children"] = [msg]  # cannot be overwritten
+        kwargs["_metadata"] = {"id_": id_}
 
-        # create the alert
         alert = v.Alert(**kwargs)
 
-        # add the alert to the app
-        self.content.children = [alert] + self.content.children.copy()
+        # Verify if alert is already in the app.
```

Comment:
If you click several times the `theme` or `locale` button, you'll end with a lot of banners with the same info.
I've added this verification to avoid multiple aggregations, what do you think? perhaps there is a cleaner way to avoid those nested if's-

Reply:
Agreed, note that if we do that then "locale" and "theme" need to become protected `id_` for Tiles (as they are also children of app.) 

I'm trying something with `next` and a single `try ... except` let's see how it goes

Reply:
What do you mean with "become protected `id_`"?, Tile cards have different metadata key in the dictionary so I don't see how they can be affected.

Reply:
true, I'm positive to COVID I'm less sharp than usual but I would like to finish this PR before going to bed to hibernate 😷

Reply:
any better since last push ?